src/App.php

```diff
@@ -78,17 +79,24 @@ private function getEventManager()
     /**
      * Application execution.
      *
-     * @param mixed|null $request  Representation of an outgoing, client-side request.
-     * @param mixed|null $response Representation of an incoming, server-side response.
+     * @param mixed|null $request  Representation of an outgoing,
+     *  client-side request.
+     * @param mixed|null $response Representation of an incoming,
+     *  server-side response.
      *
      * @return mixed
      */
     public function run($request = null, $response = null)
     {
-        $request = $request ?: ServerRequestFactory::fromGlobals();
-        $response = $response ?: new Response();
-
-        $event = new HttpFlowEvent('bootstrap', $request, $response);
+        $event = $this->getContainer()->get("http_flow_event");
```

Comment:
see https://github.com/pennyphp/penny/blob/master/src/App.php#L48-L51 , `getContainer()` only return `container`, and calling internally should not need to call getter, just call `$this->container` direclty